common/video/image_compare.cc

```diff
@@ -0,0 +1,15 @@
+#include "opencv2/opencv.hpp"
```

Comment:
this works, but typically you include the header file (in this case `image_compare.hh`) to pick up all of the interface related header files you'll need.